src/lib.rs

```diff
@@ -159,6 +162,12 @@ impl Toasts {
         self
     }
 
+    /// Margin or distance from screen to toasts' bounding boxes
```

Comment:
This comment looks confusing, how is it related to shadows?

Reply:
I missed this one! Fixed!